### PR TITLE
Made the user add button consistent with others

### PR DESCRIPTION
**Pull request by siva-balan**
## Proposed Changes
Made the user add button consistent with others
- Before:
image

- After:
image

- Made the new user add button consistent with other page buttons like create assets etc, to give it a neat look and to reduce unnecessary space.

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by siva-balan** [comment]
@nihal467 @rithviknishad  if you like this UI update please merge this.

**Review by rithviknishad — APPROVED** [review]

**Review by rithviknishad — COMMENTED** [review]
@siva-balan could you use `ButtonV2` so that we have consistent design?

**Comment by siva-balan** [comment]
@rithviknishad please re-review this and change the label.

**Comment by nihal467** [comment]
@siva-balan 

in the normal view and the responsive view, add user button width needs to increase similar to the advanced filter button to maintain the consistency

image
image

**Comment by siva-balan** [comment]
@nihal467 the update for the view is done. please check

File: src/Components/Users/ManageUsers.tsx [hunk 1]
```diff
@@ -130,13 +130,13 @@ export default function ManageUsers() {
   );
 
   const addUser = (
-    <button
-      className="px-4 py-1 w-full md:w-auto rounded-md bg-primary-500 text-white text-lg font-semibold shadow"
+    <ButtonV2
+      className="btn btn-primary-ghost w-full"
```
Comment (rithviknishad) [inline src/Components/Users/ManageUsers.tsx]:
```suggestion
      className="w-full"
```

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**